prometheus_hardware_exporter/collector.py

```diff
@@ -5,12 +5,134 @@
 
 from prometheus_client.metrics_core import GaugeMetricFamily
 
+from .collectors.perccli import PercCLI
 from .collectors.storcli import StorCLI
 from .core import BlockingCollector, Payload, Specification
 
 logger = getLogger(__name__)
 
 
+class PowerEdgeRAIDCollector(BlockingCollector):
+    """Collector for PowerEdge RAID controller."""
+
+    perccli = PercCLI()
+
+    @property
+    def specifications(self) -> List[Specification]:
+        return [
+            Specification(
+                name="poweredgeraid_controllers",
+                documentation="Total number of controllers",
+                labels=["hostname"],
```

Comment:
I actually drop it in my revamp of MegaRAID collector because there's `instance` label given by prometheus, which is the same as `hostname`

Reply:
ack